e2e/cypress/integration/enterprise/cloud/onboarding/next_steps_tips_spec.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @onboarding @cloud_only
+
+describe('Cloud Onboarding - Next Steps Tips', () => {
+    let townSquarePage;
+    let sysadmin;
+
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            townSquarePage = `/${team.name}/channels/town-square`;
+        });
+
+        cy.apiAdminLogin().then((res) => {
+            sysadmin = res.user;
+        });
+    });
+
+    beforeEach(() => {
+        // # Login as sysadmin and set all steps to false
+        const preference = {
+            user_id: sysadmin.id,
+            category: 'recommended_next_steps',
+            value: 'false',
+        };
+        const adminSteps = [
+            'complete_profile',
+            'notification_setup',
+            'team_setup',
+            'invite_members',
+            'enter_support_email',
+            'hide',
+            'skip',
+        ];
+
+        cy.apiSaveUserPreference(adminSteps.map((step) => ({...preference, name: step})));
+        cy.visit(townSquarePage);
+
+        cy.get('.NextStepsView__button:contains(Skip Getting Started)').scrollIntoView().should('be.visible').click();
+    });
+
+    it('Show incident collaboration', () => {
+        // * Check that Next Steps page shows
+        cy.get('span').contains('Tips & Next Steps');
+
+        // * IC is visible in cloud
+        cy.findByText('Resolve incidents faster with Mattermost Incident Collaboration.').should('be.visible');
```

Comment:
@pablovelezvidal any ideas how we can verify the opposite of this?

Reply:
I didn't see anything in Cypress's API about testing for absent elements. I'm not sure, but do the `find*` functions throw an error if they fail to find anything? If so, we could have some code like:

```js
let failedToFind = false
try {
  cy.findByText("text of nonexistent element")
} catch {
  failedToFind = true
}
expect(failedToFind).toBe(true);
```

Reply:
sorry I didn't post it here, I suggested via DM to Allan that he could just use `.should('not.exist');` .